src/SelectableBox/README.md

```diff
@@ -32,13 +32,15 @@ As ``Checkbox``
   };
   
   const isInvalid = () => checkedCheeses.includes('swiss');
-
+  const isExtraSmall = useMediaQuery({maxWidth: breakpoints.extraSmall.maxWidth});
```

Comment:
nit: add space between the `{` and `}`, (e.g., `useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth })`)

Same with the other cases.

Reply:
Corrected